Dockerfile.fuzz

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.14
+
+RUN apt-get update && apt-get install -y clang
+COPY . $GOPATH/src/github.com/opencontainers/umoci
+
+RUN cd $GOPATH/src/github.com/opencontainers/umoci && \
+    go clean --modcache && \
+    go mod tidy && \
+    go mod vendor && \
+    rm -r ./vendor
+
+RUN go get -u github.com/dvyukov/go-fuzz/go-fuzz \
+	github.com/dvyukov/go-fuzz/go-fuzz-dep \
+	github.com/dvyukov/go-fuzz/go-fuzz-build
```

Comment:
Can you move this to before the `COPY` -- that way it doesn't get run each time the source directory changes.